app.js

```diff
@@ -0,0 +1,27 @@
+require('dotenv').config();
+const express = require('express');
+
+const connectDB = require('./database');
+const resources = require('./resources');
```

Comment:
Comment out line 5 and any other WIP code that crashes the server.

Reply:
Thanks for the feedback, well noted.